src/components/NetworkPanel.vue

```diff
@@ -0,0 +1,234 @@
+                  color="primary"
+                  @click="downloadItem(item)"
+                >save_alt</v-icon>
+              </v-btn>
+            </v-list-item-action>
+            <v-list-item-action class="mx-0 my-0"
+              @click.prevent
+              v-if="hover"
+            >
+              <v-btn icon>
+                <v-icon
+                  color="red accent-3"
+                  @click="deleteItem(item)"
+                >delete</v-icon>
+              </v-btn>
+            </v-list-item-action>
+          </v-list-item>
+        </v-hover>
+      </v-list-item-group>
+    </template>
+    <div
+      v-else
+      class="ws-detail-empty-list"
+    >
+      <v-icon color="blue lighten-1">info</v-icon> There's nothing here yet...
+    </div>
+  </v-list>
+</template>
+
+<script lang="ts">
+import Vue, { PropType } from 'vue';
+import DeleteGraphDialog from '@/components/DeleteGraphDialog.vue';
+import GraphDialog from '@/components/GraphDialog.vue';
+import DownloadDialog from '@/components/DownloadDialog.vue';
+
+export default Vue.extend({
+  name: 'GraphPanel',
+
+  components: {
+    DeleteGraphDialog,
+    GraphDialog,
+    DownloadDialog,
+  },
+
+  props: {
+    workspace: {
+      type: String as PropType<string>,
+      required: true,
+    },
+
+    items: {
+      type: Array as PropType<string[]>,
+      required: true,
+    },
+
+    nodeTables: {
+      type: Array as PropType<string[]>,
+      required: true,
+    },
+
+    edgeTables: {
+      type: Array as PropType<string[]>,
+      required: true,
+    },
+  },
+
+  data() {
+    interface CheckboxTable {
+      [index: string]: boolean;
+    }
+
+    return {
+      checkbox: {} as CheckboxTable,
+      singleSelected: null as string | null,
+      deleterDialog: false,
+      downloaderDialog: false,
+    };
+  },
+
+  computed: {
+    checked(): string[] {
+      return Object.keys(this.checkbox)
+        .filter((d) => !!this.checkbox[d]);
+    },
+
+    selection(): string[] {
+      const {
+        singleSelected,
+        checked,
+      } = this;
+
+      let result: string[] = [];
+
+      if (singleSelected !== null) {
+        result = result.concat([singleSelected]);
+      } else {
+        result = result.concat(checked);
+      }
+
+      return result;
+    },
+
+    anySelected(): boolean {
+      return this.selection.length > 0;
+    },
+  },
+
+  methods: {
+    deleteItem(item: string) {
+      this.singleSelected = item;
+      (this.$refs.deleter as any).dialog = true;
+    },
+
+    downloadItem(item: string) {
+      this.singleSelected = item;
+      (this.$refs.downloader as any).dialog = true;
+    },
+
+    clearCheckboxes() {
+      Object.keys(this.checkbox).forEach((key) => {
+        this.checkbox[key] = false;
+      });
+    },
+
+    cleanup(deleted?: string[]) {
+      const {
+        singleSelected,
+      } = this;
+
+      if (singleSelected !== null) {
+        this.singleSelected = null;
+      }
```

Comment:
Same as the other.

Reply:
See 14dcbf1dbaf16da19aa4551782d5a6419fad6314.